Makefile

```diff
@@ -1,5 +1,5 @@
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.30
+ENVTEST_K8S_VERSION = 1.31
```

Comment:
Do we want to bump it to 1.32 since we support it ?
https://github.com/kubernetes-sigs/jobset/blob/main/go.mod#L12

Reply:
I tried that but 1.32 seems to not work yet for us.

Reply:
I can bump it here and try to fix it. probably best.  

Bumping from 1.30 to 1.32 seemed like a big jump.

Reply:
Do you want to do that as a followup ?
I am fine migrating to 1.31 first.